sdk/core/core-lro/review/core-lro.api.md

```diff
@@ -42,6 +57,13 @@ export interface LroResponse<T> {
     rawResponse: RawResponse;
 }
 
+// @public
+export interface OperationState<TResult> {
+    error?: Error;
+    result?: TResult;
+    status: "notStarted" | "running" | "succeeded" | "canceling" | "canceled" | "failed";
```

Comment:
Is `canceling` actually supported? IIRC the REST board described this as a closed set and `canceling` wasn't in it.

Also just want to make sure we don't need to account for both spellings of "cancelled"/"canceled".

Reply:
Is there a reference for that? IIRC, the closed set thing is for the terminal states set which should be `succeeded`, `canceled`, and `failed`. See 
![z5yVzle - Imgur](https://user-images.githubusercontent.com/6074665/184192038-86b82b84-55e1-4f18-9be4-58c37303a367.png)
https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md

Reply:
Is there a reference for that? IIRC, the closed set thing is for the terminal states set which should be `succeeded`, `canceled`, and `failed`. See 
![z5yVzle - Imgur](https://user-images.githubusercontent.com/6074665/184192038-86b82b84-55e1-4f18-9be4-58c37303a367.png)
https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md

Reply:
> Also just want to make sure we don't need to account for both spellings of "cancelled"/"canceled".

The poller implementation handles the spelling variation issue under the hood so both spelling don't need to be handled by client code.

Reply:
I will mark this as resolved but let me know if you disagree.

Reply:
I just think it might be strange if we export this in the constraint for OperationState but no poller is actually capable of producing it.

Reply:
It's neat that the poller automatically converts the spelling though --- didn't know that.

Reply:
I guess one way of looking at this is that whether a client would be interested in handling a "canceling" state. I am struggling to think of a reason why. I removed it in [5baa826](https://github.com/Azure/azure-sdk-for-js/pull/22753/commits/5baa826678bbc2e36b4f7052d9e7775294eab9df).